cmd/dnet/dnet.go

```diff
@@ -106,7 +106,7 @@ type dnetConnection struct {
 }
 
 func (d *dnetConnection) dnetDaemon() error {
-	controller, err := libnetwork.New()
+	controller, err := libnetwork.New("")
```

Comment:
Shouldn't this be trying to pass the default `/etc/default/libnetwork.toml`? Also shouldn't we add a config option for folks to add a different config file location